oc-chef-pedant/spec/api/user_spec.rb

```diff
@@ -185,19 +185,19 @@
 
             let(:filtered_users_body) do
               { username => "#{request_url}/#{username}",
-                username_two => "#{request_url}/#{username_two}",
               }
             end
 
             before :each do
-              @user2 = platform.create_user(username_two, user_options_two)
+              begin
+                platform.create_user(username_two, user_options_two)
+              rescue StandardError => e  
```

Comment:
With this rescue in place, we aren't ever verifying what actually happens on duplicate case-insensitive email address. 

Given that this is a change in functionality, we should also add a specific example that verifies the exception we expect comes from creating a user with a duplicate email address; doing that would make it safe to do rescue this here. 

 In 'when there's another user' context, something like
```
it "raises  $exception-type when trying to create that user" do 

... 

end

Reply:
Added test for convering the above suggested case